src/main/java/uk/ac/cam/cl/dtg/isaac/dto/users/UserSummaryWithEmailAddressAndGenderDTO.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2017 James Sharkey
+ * <br>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * <br>
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <br>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.cam.cl.dtg.isaac.dto.users;
+
+/**
+ * View of a User Summary object, which contains additional information (email).
+ * Usually used where a sharing relationship has or will be created.
+ */
+public class UserSummaryWithEmailAddressAndGenderDTO extends UserSummaryDTO {
+  private String email;
+  private String gender;
+
+  /**
+   * UserSummaryDTO.
```

Comment:
All the comments in this class are either incorrect or they don't provide any useful information, so I would get rid of all of them.